README.md

```diff
@@ -9,8 +9,15 @@ In the case of the onprem provider, `dcos-launch` provides the capability to cre
 
 The primary purpose of `dcos-launch` is to provide a turn-key deployment experience across a wide variety of configurations without having to handle any of the lower-level cloud APIs. As such, `dcos-launch` includes a [command line interface](dcos_launch/README.md) and is packaged as a pyinstaller [binary](https://downloads.dcos.io/dcos-launch/bin/linux/dcos-launch) for easy portability.
 
+## Installation
+```
+git clone https://github.com/dcos/dcos-launch.git
+cd dcos-launch
+pip3 install -r requirements.txt && pip3 install -e .
```

Comment:
I would strongly advise against doing any work where you install to your local python. This is especially relevant when you have requirements that are pinned to git SHA as that SHA is lost after download. Thus, if you move that pin you will need to uninstall that development library from local host. With virtualenv, you can blast the whole thing away and recreate without issue.

Reply:
virtualenv was an assumption on my part, I have no objection to referencing it explicitly.